src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -330,11 +330,13 @@ HRESULT STDMETHODCALLTYPE CorProfiler::ModuleLoadFinished(ModuleID module_id,
       "mscorlib"_W,
       "netstandard"_W,
       "Datadog.Trace"_W,
+      "Datadog.Trace.AspNet"_W,
       "Datadog.Trace.ClrProfiler.Managed"_W,
-      "MsgPack"_W,
-      "MsgPack.Serialization.EmittingSerializers.GeneratedSerealizers0"_W,
-      "MsgPack.Serialization.EmittingSerializers.GeneratedSerealizers1"_W,
-      "MsgPack.Serialization.EmittingSerializers.GeneratedSerealizers2"_W,
+      "Datadog.Trace.ClrProfiler.Managed.Loader"_W,
```

Comment:
Should we add `Datadog.Trace.ClrProfiler.Managed.Core`, too?

Reply:
Addressed in 60ad318fb3fbe2d48b3477163178326edb021cc3